partner_aging/wizard/customer.py

```diff
@@ -0,0 +1,178 @@
+                   FROM account_move_line lt LEFT JOIN account_invoice inv on lt.move_id = inv.move_id   
+                ) DaysDue       
+                ON DaysDue.id = aml.id
+                LEFT JOIN account_invoice as ai ON ai.move_id = aml.move_id
+                WHERE
+                aml.user_type_id in (select id from account_account_type where type = 'receivable')
+                and aml.date <= '%s'
+                and aml.amount_residual != 0
+                GROUP BY aml.partner_id, aml.id, ai.number, days_due, ai.user_id, ai.id
+              """ % (age_date, age_date, age_date, age_date, age_date,  # noqa
+                     age_date, age_date, age_date, age_date, age_date,
+                     age_date, age_date, age_date, age_date)
+
+        tools.drop_view_if_exists(self.env.cr, '%s' %
+                                  (self._name.replace('.', '_')))
+        self.env.cr.execute("""CREATE OR REPLACE VIEW %s AS (%s)""" %
+                            (self._name.replace('.', '_'), query))
+
+    @api.multi
+    def docopen(self):
+        """
+        @description  Open document (invoice or payment) related to the
+                      unapplied payment or outstanding balance on this line
+        """
+
+        models = self.env['ir.model.data']
+        # Get this line's invoice id
+        inv_id = self.invoice_id.id
+
+        # If this is an unapplied payment (all unapplied payments hard-coded to
+        #  -999), get the referenced voucher
+        if inv_id == -999:
+            payment = self.env['account.voucher']
+            # Get referenced customer payment
+            # (invoice_ref field is actually a payment for these)
+            voucher_id = payment.search([('number', '=', self.invoice_ref)])[0]
+            view = models.xmlid_to_object('account_voucher.view_voucher_form')
+            # Set values for form
+            view_id = view and view.id or False
+            name = 'Customer Payments'
+            res_model = 'account.voucher'
+            ctx = "{}"
+            doc_id = voucher_id
+
+        # Otherwise get the invoice
+        else:
+            view = models.xmlid_to_object('account.invoice_form')
+            view_id = view and view.id or False
+            name = 'Customer Invoices'
+            res_model = 'account.invoice'
+            ctx = "{'type':'out_invoice'}"
+            doc_id = inv_id
+
+        if not doc_id:
+            return {}
+
+        # Open up the document's form
+        return {
+            'name': _(name),
+            'view_type': 'form',
+            'view_mode': 'form',
+            'view_id': [view_id],
+            'res_model': res_model,
+            'context': ctx,
+            'type': 'ir.actions.act_window',
+            'nodestroy': True,
+            'target': 'current',
+            'res_id': doc_id,
+        }
+
+    partner_id = fields.Many2one('res.partner', u'Partner', readonly=True)
```

Comment:
[Field declarations should be at top.](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#models)

Also, please remove all the forced unicodes